sql/core/src/test/resources/sql-tests/results/operators.sql.out

```diff
@@ -316,3 +316,25 @@ select cot(-1)
 struct<COT(CAST(-1 AS DOUBLE)):double>
 -- !query 37 output
 -0.6420926159343306
+
+-- !query 38
+select bit_length(`abc`)
+-- !query 38 schema
+struct<BIT_LENGTH(abc):int>
+-- !query 38 output
+24
+
+-- !query 39
+select character_length(`abc`)
+-- !query 39 schema
+struct<CHARACTER_LENGTH(abc):int>
+-- !query 39 output
+3
+
+-- !query 40
+select octet_length(`abc`)
```

Comment:
[These test cases](https://github.com/apache/spark/pull/18046/files#diff-e4663e57952b37150642b33b998715a8R527) handles the case.